docs/2.8.0/docs/daml-finance/packages/interfaces/changelogs/daml-finance-interface-instrument-equity.rst

```diff
@@ -4,14 +4,27 @@
 Daml.Finance.Interface.Instrument.Equity - Changelog
 ####################################################
 
+Version 0.4.0
+*************
+
+- Update of SDK version and dependencies.
+
+- The `InstrumentKey` is extended by the `HoldingStandard` field.
+
+- The `Remove` choice was removed from the `Factory` (it is newly part of the `Base` instrument
+  interface).
+
+- Renamed the `F` type synonym to `I`.
+
 Version 0.3.0
 *************
 
 - Update of SDK version and dependencies
 
-- The `Create` choice on the instrument factory returns the corresponding interface (rather than the base instrument interface)
+- The `Create` choice on the instrument factory returns the corresponding interface (rather than the
```

Comment:
Here's another section where the list items don't have terminal punctuation; up to you whether to add it